src/electron/views/automated-checks/automated-checks-view.scss

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+.automated-checks-view {
+    main {
+        margin: 16px 24px;
```

Comment:
Consider using the named versions (`margin-top`, etc) for setting multiples like this; harder to make mistakes/miss stuff in review if that's our default practice

Reply:
updated